examples/examples-cpu/nyc-taxi-snowflake/xgboost-dask.ipynb

```diff
@@ -27,8 +25,27 @@
    "outputs": [],
    "source": [
     "import os\n",
-    "import numpy as np\n",
+    "import warnings\n",
+    "\n",
+    "import cloudpickle\n",
+    "import dask.dataframe as dd\n",
+    "import dask_xgboost\n",
+    "import snowflake.connector\n",
+    "\n",
+    "from dask import delayed\n",
```

Comment:
I think we ended up on `import dask` use `@dask.delayed` while you are in here tidying up. I also think it would make the most sense to me to group the dask imports more.